CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ All notable changes to [camunda-bpmn-js](https://github.com/camunda/camunda-bpmn
 
 ___Note:__ Yet to be released changes appear here._
 
+## 2.6.1
+
+* `DEPS`: update to `@bpmn-io/element-template-icon-renderer@0.5.1`
+
+### Key changes in Modeling
+
+* `FIX`: intermediate catch and throw events with icons render successfully ([#16](https://github.com/bpmn-io/element-template-icon-renderer/pull/16))
```

Comment:
#16 links to another repo - lets align it with the Changelog style of this repo

```suggestion
* `FIX`: intermediate catch and throw events with icons render successfully ([bpmn-io/element-template-icon-renderer#16](https://github.com/bpmn-io/element-template-icon-renderer/pull/16))
```

Reply:
Ah, didn't know about this new convention. Makes sense to me!